_data/navbar.yml

```diff
@@ -7,37 +7,19 @@ menu:
         show_in_menu: false
         show_in_footer: false
     -
-        text: Feedback Program
+        text: About
+        id: about
+        href: /about
+        permalink: /about/
+        show_in_menu: true
+        show_in_footer: false
+    -
+        text: Services
         id: services
         href: /services
         permalink: /services/
         show_in_menu: true
         show_in_footer: false
-        children:
-            -
```

Comment:
removing these will break the sidenav. 

if you don't want them, be sure to change the page layout.

Reply:
That's why this was WIP :)

Reply:
Yep, WIP